pkg/vm/slot.go

```diff
@@ -1,6 +1,8 @@
 package vm
 
 import (
+	"encoding/json"
+
```

Comment:
beep

Reply:
It is intentionally, otherwise, linter doesn't like it.